### ISSUE TITLE
item/mycelium_wall not rendering

### ISSUE THREAD

**Issue by puggan** [ConsistencyPlus/ConsistencyPlus#42]
**Describe the bug**
The mycelium wall item have no texture.

Thinks it's from this line:
https://github.com/ConsistencyPlus/ConsistencyPlus/blob/43918ad43e847c4ff3049d698963f6b263e8b770/src/main/resources/assets/consistency_plus/models/item/mycelium_wall.json#L4-L4

**Versions**
Minecraft version: 1.17.1
C+ version: 0.4.0-RC1-b283
Fabric API version: 0.11.6

**Other mods**
Fabric API 0.36.1+1.17.1

**Logs**
```
[Worker-Main-10/ERROR]: Using missing texture, unable to load minecraft:textures/block/mycelium.png : java.io.FileNotFoundException: minecraft:textures/block/mycelium.png
```

**To Reproduce**
Steps to reproduce the behavior. Format it as a numbered list of steps.

1. Start a world, in creative (or with cheats enabled)
2. Open the creative inventory at "Consistency+ Misc"-tab
3. Scroll down to Mycelium wall (last row)

**Expected behavior**
Render the mycelium wall

**Screenshots**
![2021-07-12_15 11 39](https://user-images.githubusercontent.com/866668/125293284-8d5d6200-e323-11eb-821f-0c583281e92c.png)

**Comment by Siuolthepic** [comment]
I'll fix it before 0.4 full

**Comment by Siuolthepic** [comment]
Fixed, am closing.
